modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

```diff
@@ -3,6 +3,7 @@
 :msoffice-online-server-url: https://www.microsoft.com/en-us/microsoft-365/blog/2016/05/04/office-online-server-now-available/
 :office365-url: https://products.office.com/en-us/business/office 
 :wopi-protocol-url: https://wopi.readthedocs.io/en/latest/
+:officeonline-server: https://docs.microsoft.com/de-de/officeonlineserver/deploy-office-online-server
```

Comment:
I know it is a pain, but you need to name it with ending -url.
This helps to distinguish on the fly if the name references a url or not.

Reply:
No worries. I did not see the other links having the -url ending.

added